foundation-extensions/identity-for-edge-network/api-reference.md

```diff
@@ -141,81 +194,96 @@ Identity.getIdentities { (identityMap, error) in
   }
 }
 ```
+
+**Objective-C**
+
+**Syntax**
+
+```objectivec
++ (void) getIdentities:^(AEPIdentityMap * _Nullable map, NSError * _Nullable error)completion
+```
+
+**Examples**
+
+```objectivec
+[AEPEdgeIdentity getIdentities:^(AEPIdentityMap *map, NSError * error) {   
+    // handle the retrieved identities here    
+}];
+```
+
 {% endtab %}
 {% endtabs %}
 
-## updateIdentities
+
+## registerExtension
 
 {% tabs %}
 {% tab title="Android" %}
-### updateIdentities
-
-Update the currently known identities within the SDK. The Identity extension will merge the received identifiers with the previously saved ones in an additive manner, no identities are removed from this API.
+### registerExtension
 
-Identities with an empty _id_ or _namespace_ are not allowed and are ignored.
-
-Updating identities using a reserved namespace is not allowed using this API. The reserved namespaces are:
-
-* ECID
-* IDFA
-* GAID
+Registration occurs by passing identity for edge network extension to the MobileCore.registerExtensions API.
```

Comment:
Capitalize: "Identity for Edge Network extension"
Also, should we hyperlink "Mobilecore.registerExtensions" to its API reference?
Plus, as in my previous comment, move this line out of the tab group.